src/pip/_internal/commands/configuration.py

```diff
@@ -225,6 +226,10 @@ def open_in_editor(self, options: Values, args: List[str]) -> None:
 
         try:
             subprocess.check_call([editor, fname])
+        except FileNotFoundError as e:
+            if WINDOWS:
+                e.filename = editor
```

Comment:
```suggestion
            if not e.filename:
                e.filename = editor
```

We don’t need to make this specific to Windows. This is better IMO.

Reply:
Ok.
Should I add a test for this?

Reply:
Added.